src/test/java/study/SayMethodMatcher.java

```diff
@@ -0,0 +1,13 @@
+package study;
+
+import java.lang.reflect.Method;
+
+public class SayMethodMatcher implements MethodMatcher {
+    @Override
+    public boolean matches(Method m, Class targetClass, Object[] args) {
+        if(m.getName().startsWith("say")) {
+            return true;
+        }
+        return false;
```

Comment:
이렇게 해보셔도 좋을 것 같아요 :)
```suggestion
        return m.getName().startsWith("say");
```